pkg/cloud/gcp/error_test.go

```diff
@@ -31,5 +31,5 @@ func TestErrorBehaviour(t *testing.T) {
 	wrap1 := errors.Wrap(orig, "wrap1")
 	wrap2 := errors.Wrap(wrap1, "wrap2")
 	assert.Equal(t, "wrap1: googleapi: Error 403: ACCESS DENIED. ALL YOUR BASE ARE BELONG TO US", wrap1.Error())
-	assert.Equal(t, "wrap2: wrap1: googleapi: Error 403: ACCESS DENIED. ALL YOUR BASE ARE BELONG TO US", wrap2.Error())
+	assert.Equal(t, "wrap2: wrap1: grpc: ACCESS DENIED. ALL YOUR BASE ARE BELONG TO US [code 2/Unknown]", wrap2.Error())
```

Comment:
This change was caused by https://github.com/googleapis/gax-go/commit/043b73437a240a91229207fb3ee52a9935a36f23

Generally, I'm OK with this for now since we clearly still get the underlying error message.  

However, I think as a follow up, over in `grpcSpecialCasePrintFn` we should perhaps just do an explicit type check for the APIError type and opt out of the special printing.

Reply:
TFTR! I could do that here.. but do we want to import `gax-go` from `grpcutil`?

Reply:
It's a good point.  Some options that I haven't thought through:

1. Hacky (but maybe not completely horrible because we have some tests here?): look for `googleapi:` explicitly?
2. I'm loathe to add yet-another-feature to the errors package. But perhaps we could have a new API for registering special printers that could take a type name.  We would consult type-specific special printers first and match the type name based on reflection.

Reply:
I'll leave it as is for now. As you said, it's not a huge deal since we still a useful message. I don't really understand why the message is different only when wrapping twice and not when wrapping once.